ci/gpu/build.sh

```diff
@@ -176,6 +176,19 @@ else
         ${gt} --gtest_output=xml:"$WORKSPACE/test-results/"
     done
 
+    if [[ "$COMPUTE_SANITIZER_ENABLE" == "1" ]]; then
+        gpuci_logger "Memcheck on GoogleTests with rmm_mode=cuda"
+        export GTEST_CUDF_RMM_MODE=cuda
+        COMPUTE_SANITIZER_CMD="compute-sanitizer --tool memcheck --log-file" #  $WORKSPACE/test-results/cudamemcheck-%q{test_name}.log
```

Comment:
Did we intend to remove that `$WORKSPACE` bit?
```suggestion
        COMPUTE_SANITIZER_CMD="compute-sanitizer --tool memcheck --log-file"
```
Or is it to illustrate how `$COMPUTE_SANITIZER_CMD` is completed when used?

Reply:
Also, update Copyright year?

Reply:
updated.